### ISSUE TITLE
I got Warning pytorch not support instance-level annotations.

### ISSUE THREAD

**Issue by RBERLOP** [ACEsuit/mace#573]
**Describe the bug**

At the begining of a training run, I got this Warning message spamming:

```
2024-08-28 16:40:45.227 INFO: Hidden irreps: 256x0e

/home/username/.local/lib/python3.9/site-packages/torch/jit/_check.py:178: UserWarning: The TorchScript type system doesn't support instance-level annotations on empty non-base types in `__init__`. Instead, either 1) use a type annotation in the class body, or 2) wrap the type in `torch.jit.Attribute`.
 
```
After that warning, it describes the parameters, I do not printed them just to not overextend this message.  
The calculations seemed to continue normally but I then at the first 2 loss steps the RMSE in forces and energies dropped to 0

```
Parameter Group 4
    amsgrad: True
    betas: (0.9, 0.999)
    capturable: False
    differentiable: False
    eps: 1e-08
    foreach: None
    fused: None
    initial_lr: 0.01
    lr: 0.01
    maximize: False
    name: readouts
    swa_lr: 0.001
    weight_decay: 0.0
)
2024-08-28 16:40:46.143 INFO: Using gradient clipping with tolerance=10.000
2024-08-28 16:40:46.143 INFO: Started training
2024-08-28 16:40:50.776 INFO: Epoch None: loss=0.0000, RMSE_E_per_atom=4.0 meV, RMSE_F=15.4 meV / A
2024-08-28 16:44:18.919 INFO: Epoch 0: loss=0.0000, RMSE_E_per_atom=0.0 meV, RMSE_F=0.0 meV / A
2024-08-28 16:47:55.897 INFO: Epoch 1: loss=0.0000, RMSE_E_per_atom=0.0 meV, RMSE_F=0.0 meV / A
2024-08-28 16:51:56.137 INFO: Epoch 2: loss=0.0000, RMSE_E_per_atom=0.0 meV, RMSE_F=0.0 meV / A
2024-08-28 16:57:27.578 INFO: Epoch 3: loss=0.0000, RMSE_E_per_atom=0.0 meV, RMSE_F=0.0 meV / A
```
I launched mace using the following flags:
```
python3 run_train.py \
     --name="solid" \
     --train_file="dataset_1022.xyz" \
     --valid_fraction=0.05 \
     --E0s="average"            \
     --energy_key="REF_energy"  \
     --forces_key="REF_forces" \
     --stress_key="REF_stress"  \
     --model="MACE"  \
     --num_interactions=2  \
     --max_ell=2  \
     --hidden_irreps="256x0e"  \
     --num_cutoff_basis=5  \
     --correlation=2  \
     --r_max=3.0  \
     --batch_size=3  \
     --valid_batch_size=3  \
     --eval_interval=1  \
     --max_num_epochs=50  \
     --start_swa=15  \
     --swa_energy_weight=1000  \
     --ema  \
     --ema_decay=0.99  \
     --amsgrad  \
     --error_table="PerAtomRMSE"  \
     --default_dtype="float32"  \
     --swa  \
     --device=cpu  \
     --seed=1234

```

I don't know if this drop of the values of RMSE is related with the Pytorch warning at the begenning (Technical issue) , or I just need to adjust the parameters when launching the program (Scientific issue).

My sistem is a surface supported metal particle interacting with water molecules

**Comment by ilyes319** [comment]
Hey,

What branch are you using? Also can you please link you all log file.
Your problem is not raleted to the warning. Please make sure that you are using the right keys for your data. Did you name your forces and energies "REF_forces" and "REF_energy" in your xyz file?

**Comment by RBERLOP** [comment]
[titania1_run-1234.log](https://github.com/user-attachments/files/16786574/titania1_run-1234.log)

Log file uploaded. 

Thank you very much for your reply, I am using version 0.3.6. And thank you very much for your suggestions, it worked! 
In the input file I have energies printed as "dft_energies" and forces as "forces". I changed the flag REF_energy and the REF_forces to dft_energies and forces, respectively. 

Now the training is running with no problem but I notice another spamming WARNING about the ase caclulator has not attribute 'forces'- Reading the documentation it suggests to overwrite energy and forces to REF_energy and REF_forces (which I did, partially) to avoid problems with ASE Calculator. 

Shall I change the label forces?

**Comment by ilyes319** [comment]
Great! Yes exactly you should change the label, because with newer ASE version it is no longer safe to access forces under the "forces", it is reserved to the calculator.
I will close the issue.

**Comment by RBERLOP** [comment]
Ok, wait, but the warning is still spamming ata the begining of the calculation:

/home/rbergua/.local/lib/python3.9/site-packages/torch/jit/_check.py:178: UserWarning: The TorchScript type system doesn't support instance-level annotations on empty non-base types in `__init__`. Instead, either 1) use a type annotation in the class body, or 2) wrap the type in `torch.jit.Attribute`.
  warnings.warn(

How can Iwe deal with this¿

Thank you very much for your help

**Comment by ilyes319** [comment]
nothing to do.
